README.md

```diff
@@ -30,31 +30,45 @@ datalad get -s osf-storage neuronets/ams/0.1.0/meningioma_T1wc_128iso_v1.h5
 ```
 
 ```
-datalad get -s neuronets/braingen
+datalad get -s osf-storage neuronets/braingen
 ```
 
-## Loading the models
+## Loading models for training
 
-The models can be loaded with `tf.keras.models.load_model`. 
+All models are available for re-training or transfer learning purposes except the **kwyk** model, which its weights are not available in tf2 keras format (We are working to make it available in near future). These models can be loaded with `tf.keras.models.load_model`.
 
 ```
-import nobrainer
 import tensorflow as tf
 
-model = tf.keras.models.load_model("neuronets/kwyk/0.4.1/all_50_bvwn_multi_prior")
-model.predict(...)
+model = tf.keras.models.load_model("neuronets/brainy/0.1.0/brain-extraction-unet-128iso-model.h5")
+model.fit(...)
 ```
 
-The `h5` files (brainy and ams) contain weights that can be loaded onto an instantiated architecture.
+You can see a transfer learning example [here](https://github.com/neuronets/nobrainer/blob/master/guide/transfer_learning.ipynb), and an example of brain MRI generation using **braingen** models can be find [here](https://github.com/neuronets/nobrainer/blob/master/guide/train_generation_progressive.ipynb).
+
+
+## Using models for inference
+
+You can use [_Nobrainer_](https://github.com/neuronets/nobrainer) toolbox for inference.
 
 ```
 import nobrainer
-import tensorflow as tf
+from nobrainer.volume import standardize_numpy
```

Comment:
i don't think this needs `standardize_numpy`. the tensorflow op should be sufficient.

```suggestion
from nobrainer.volume import standardize
```